text/0000-deprecate-tryinvoke.md

```diff
@@ -0,0 +1,91 @@
+- Start Date: 2020-10-02
```

Comment:
Can you rename this file to:

`text/0673-deprecate-tryinvoke.md` (getting rid of `0000` in favor of `0673` now taht we know the PR number)